java/alizer-api/src/main/resources/languages.yml

```diff
@@ -1904,6 +1904,10 @@ Go:
   codemirror_mode: go
   codemirror_mime_type: text/x-go
   language_id: 132
+  configuration_files:
```

Comment:
Not sure having this extra information in this file is a good idea as it will make it difficult to sync with upstream. Maybe a separate file would be better for maintenance